doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -178,6 +178,12 @@ Bluetooth samples
 Bluetooth mesh samples
 ----------------------
 
+* :ref:`bluetooth_mesh_light_switch`
+
+  * Added:
+
+    * Support for running the light switch as a Low Power Node.
```

Comment:
```suggestion
    * Support for running the light switch as a Low Power node.
```
This one slipped through... So, if you are doing any additional changes to the docs anyway, please fix this one as well :)